src/utils.ts

```diff
@@ -342,7 +342,7 @@ export const withBatchedUpdates = <
 export const nFormatter = (num: number, digits: number): string => {
   const si = [
     { value: 1, symbol: "b" },
-    { value: 1e3, symbol: "k" },
+    { value: 1e3, symbol: "K" },
     { value: 1e6, symbol: "M" },
     { value: 1e9, symbol: "G" },
```

Comment:
Let's change all of them to include B: https://en.wikipedia.org/wiki/Kilobyte

Reply:
Do I have to Change like this?
![2020-12-18_01-59](https://user-images.githubusercontent.com/24036721/102540849-a7a8c780-40d5-11eb-803d-205cd50ec75c.png)

Reply:
Yeah.. let's got with SI :)

Reply:
I have included B in SI symbols.